src/Tools/RepoUtil/ConflictingPackagesException.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace RepoUtil
+{
+    internal class ConflictingPackagesException : Exception
```

Comment:
That would be fine.  In general I favor pokemon exception handling: gotta catch them all.  The .NET ecosystem makes it hard / impossible to do fine grained exception handling.